doc/source/ray-air/getting-started.rst

```diff
@@ -32,72 +32,103 @@ To get started, install Ray AIR via ``pip install -U "ray[air]"``
 .. tabbed:: XGBoost
 
-    Preprocess your data with a ``Preprocessor``.
 
     .. literalinclude:: doc_code/xgboost_starter.py
         :language: python
         :start-after: __air_xgb_preprocess_start__
         :end-before: __air_xgb_preprocess_end__
 
-    Train a model with an ``XGBoostTrainer``.
+.. tabbed:: Pytorch
 
-    .. literalinclude:: doc_code/xgboost_starter.py
+    .. literalinclude:: doc_code/pytorch_tabular_starter.py
         :language: python
-        :start-after: __air_xgb_train_start__
-        :end-before: __air_xgb_train_end__
+        :start-after: __air_pytorch_preprocess_start__
+        :end-before: __air_pytorch_preprocess_end__
+
+.. tabbed:: Tensorflow
+
+    .. literalinclude:: doc_code/tf_tabular_starter.py
+        :language: python
+        :start-after: __air_tf_preprocess_start__
+        :end-before: __air_tf_preprocess_end__
 
+Train a model with a ``Trainer``.
 
-    Use the trained model for batch prediction with a ``BatchPredictor``.
+.. tabbed:: XGBoost
 
     .. literalinclude:: doc_code/xgboost_starter.py
         :language: python
-        :start-after: __air_xgb_batchpred_start__
-        :end-before: __air_xgb_batchpred_end__
-
+        :start-after: __air_xgb_train_start__
+        :end-before: __air_xgb_train_end__
 
 .. tabbed:: Pytorch
 
-    Create your dataset.
+    .. literalinclude:: doc_code/pytorch_tabular_starter.py
+        :language: python
+        :start-after: __air_pytorch_train_start__
+        :end-before: __air_pytorch_train_end__
+
+.. tabbed:: Tensorflow
 
-    .. literalinclude:: doc_code/pytorch_starter.py
+    .. literalinclude:: doc_code/tf_tabular_starter.py
         :language: python
-        :start-after: __air_pytorch_preprocess_start__
-        :end-before: __air_pytorch_preprocess_end__
+        :start-after: __air_tf_train_start__
+        :end-before: __air_tf_train_end__
+
+Find the best hyperparameters with a ``Tuner``.
 
-    Train a model with a ``TorchTrainer``.
+.. tabbed:: XGBoost
 
-    .. literalinclude:: doc_code/pytorch_starter.py
+    .. literalinclude:: doc_code/xgboost_starter.py
         :language: python
-        :start-after: __air_pytorch_train_start__
-        :end-before: __air_pytorch_train_end__
+        :start-after: __air_xgb_tuner_start__
+        :end-before: __air_xgb_tuner_end__
+
+.. tabbed:: Pytorch
 
+    .. literalinclude:: doc_code/pytorch_tabular_starter.py
+        :language: python
+        :start-after: __air_pytorch_tuner_start__
+        :end-before: __air_pytorch_tuner_end__
 
 .. tabbed:: Tensorflow
 
-    Create your ``Ray Dataset``.
-
-    .. literalinclude:: doc_code/tf_starter.py
+    .. literalinclude:: doc_code/tf_tabular_starter.py
         :language: python
-        :start-after: __air_tf_preprocess_start__
-        :end-before: __air_tf_preprocess_end__
+        :start-after: __air_tf_tuner_start__
+        :end-before: __air_tf_tuner_end__
+
+Use the trained model for batch prediction with a ``BatchPredictor``.
```

Comment:
The parts that are the same should not be replicated.

Reply:
Tried to reduce replication.